internal/operands/node-labeller/reconcile_test.go

```diff
@@ -0,0 +1,111 @@
+package node_labeller
+
+import (
+	"context"
+	secv1 "github.com/openshift/api/security/v1"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/errors"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	ssp "kubevirt.io/ssp-operator/api/v1alpha1"
+	"kubevirt.io/ssp-operator/internal/common"
+)
+
+var log = logf.Log.WithName("node_labeller_operand")
+
+var _ = Describe("Node Labeller operand", func() {
+	const (
+		namespace = "kubevirt"
+		name      = "test-ssp"
+	)
+
+	var (
+		request common.Request
+		operand = GetOperand()
+	)
+
+	BeforeEach(func() {
+		s := scheme.Scheme
+		Expect(ssp.AddToScheme(s)).ToNot(HaveOccurred())
+		Expect(secv1.Install(s)).ToNot(HaveOccurred())
+
+		client := fake.NewFakeClientWithScheme(s)
+		request = common.Request{
+			Request: reconcile.Request{
+				NamespacedName: types.NamespacedName{
+					Namespace: namespace,
+					Name:      name,
+				},
+			},
+			Client:  client,
+			Scheme:  s,
+			Context: context.Background(),
+			Instance: &ssp.SSP{
+				TypeMeta: meta.TypeMeta{
+					Kind:       "SSP",
+					APIVersion: ssp.GroupVersion.String(),
+				},
+				ObjectMeta: meta.ObjectMeta{
+					Name:      name,
+					Namespace: namespace,
+				},
+			},
+			Logger: log,
+		}
+	})
+
+	It("should create node labeller resources", func() {
```

Comment:
1. test IDs are missing
2. what about reconciliation after resources deletion and editing tests?